.github/workflows/e2e_testbuildbundle.yaml

```
@@ -0,0 +1,34 @@
+# this file has been generated by e2e/hack/create-e2e-workflows.sh.
+# edit there if you want to change something.
+name: E2E - TestBuildBundle
+on:
+- pull_request
+jobs:
+  e2e:
+    name: E2E - TestBuildBundle
+    runs-on: ubuntu-latest
```

is the only difference in these tests the TEST_NAME? if so i wonder if we can use a matrix?

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

```suggestion
  e2e:
    name: E2E
    runs-on: ubuntu-latest
    strategy:
      fail-fast: false
      matrix:
        test:
          - TestBuildBundle
          - TestEmbedAndInstall
          - TestInstallSingleNodeAndUpgradeToEmbed
          - TestMultiNodeInstallation
          - TestSingleNodeInstallation
          - TestTokenBasedMultiNodeInstallation
```

---

Yes, this is much better. Done.